paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetBillingConfigurationTest.kt

```diff
@@ -0,0 +1,137 @@
+package com.stripe.android.paymentsheet
+
+import androidx.compose.ui.test.hasTestTag
+import androidx.compose.ui.test.hasText
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performScrollTo
+import androidx.compose.ui.test.performTextReplacement
+import androidx.lifecycle.Lifecycle
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth
+import com.stripe.android.PaymentConfiguration
+import com.stripe.android.networktesting.NetworkRule
+import com.stripe.android.networktesting.RequestMatchers
+import com.stripe.android.networktesting.testBodyFromFile
+import com.stripe.android.paymentsheet.ui.PAYMENT_SHEET_PRIMARY_BUTTON_TEST_TAG
+import com.stripe.android.ui.core.BillingDetailsCollectionConfiguration
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+@RunWith(AndroidJUnit4::class)
+internal class PaymentSheetBillingConfigurationTest {
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<MainActivity>()
+
+    @get:Rule
+    val networkRule = NetworkRule()
+
+    @Test
+    fun testPayloadWithDefaultsAndOverrides() {
+        networkRule.enqueue(
+            RequestMatchers.method("POST"),
+            RequestMatchers.path("/v1/consumers/sessions/lookup"),
+        ) { _ -> }
```

Comment:
I think the default is a 200 with an empty body. Should we be more explicit about a failure here?

Reply:
I had no idea what to do here lol, I only added it to make the test work but I don't really care about this request. 200 with an empty body makes sense to me